src/hotspot/share/prims/whitebox.cpp

```diff
@@ -1712,8 +1712,22 @@ int WhiteBox::array_bytes_to_length(size_t bytes) {
 ///////////////
 // MetaspaceTestContext and MetaspaceTestArena
 WB_ENTRY(jlong, WB_CreateMetaspaceTestContext(JNIEnv* env, jobject wb, jlong commit_limit, jlong reserve_limit))
+  if (commit_limit % BytesPerWord != 0) {
```

Comment:
Use is_aligned() from utilities/align.hpp

Reply:
And I think you can just assert() here. If this happens, a test written by us is using the whitebox function wrong, and since its all internal, no need to propagate a java exception.